nebula-logger/plugins/async-failure-additions/plugin/flows/Nebula_Logger_Flow_Execution_Error_Event_Handling.flow-meta.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Just for consistency with other metadata filenames, could this flow be renamed to something like `LoggerFlowExecutionErrorEventHandling`?